packages/web/app/src/pages/target-laboratory.tsx

```diff
@@ -242,6 +243,40 @@ function Save(props: {
   );
 }
 
+const onModifyHeaders: ComponentProps<typeof GraphiQL>['onModifyHeaders'] = async (
+  headers = {},
+) => {
+  const { environmentVariables, logs } = await executeScript();
```

Comment:
cc @n1ru4l `executeScript` sends a message to the worker, is it fine to do it with `await`? I am not very familiar..